AdminWebsite/AdminWebsite/Dockerfile

```diff
@@ -26,4 +26,6 @@ FROM base AS final
 WORKDIR /app
 COPY --from=publish /app/publish .
 COPY --from=client /app/dist /app/ClientApp/dist
+RUN apt-get update && \
```

Comment:
Is this something needed for all apps?

Reply:
no, need to delete this actually as was for debugging